ci/prow/e2e-incluster-build

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bash
+
+set -x
+
+echo "Deploy KMMO..."
+make deploy
+
+echo "Wait until the KMMO Deployment is Available"
+echo "And Describe the Deployment / pods and get their YAML if that failed..."
+timeout 1m kubectl wait --for condition=Available deployments.apps -n kmm-operator-system kmm-operator-controller-manager || kubectl describe deployments.apps -n kmm-operator-system kmm-operator-controller-manager && kubectl get -o yaml deployments.apps -n kmm-operator-system kmm-operator-controller-manager && kubectl describe pod -n kmm-operator-system && kubectl get -o yaml pod -n kmm-operator-system
```

Comment:
```suggestion
echo "Deploy KMMO..."
make deploy
timeout 1m kubectl wait --for condition=Available deployments.apps -n kmm-operator-system kmm-operator-controller-manager
```

Reply:
By using https://github.com/kubernetes-sigs/kernel-module-management/pull/166#discussion_r1033541627, those error handling looks much cleaner and it is easier to understand what this step is supposed to do.

Reply:
Agree. Will fix.